### DISCUSSION TITLE
How to render the filled version of material symbols?

### DISCUSSION THREAD

**Discussion by Nick-Riggs** [quasarframework/quasar#17136] [General - Components / Directives / etc] [answered]
I am looking for the best (quasar) way of rendering the filled version of a material symbol for a QIcon component. 

![image](https://github.com/quasarframework/quasar/assets/210968/6e2908ee-2408-4bd5-822a-955681fa1c2e)
instead of 
![image](https://github.com/quasarframework/quasar/assets/210968/be70391a-c1b1-4a5a-a70e-736e3906c698)

Seemingly, the quasar.config file doesn't support it, nor do any props/styles I give QIcon achieve it. 

Any ideas are appreciated. Thank you.

**Comment by opsec-infosec** [comment]
I too am facing this issue, there seems to be no way to fill icons using q-icon or button.  I have to revert to material-icon instead of material-symbol-icon.

**Comment by ivanjaros** [comment]
https://quasar.dev/vue-components/icon#webfont-usage

do not use the outlined version, should be pretty straightforward.

**Reply by 7RST1** [reply]
Looks like there is no filled icon pack for material symbols, only material icons

**Comment by 7RST1** [comment]
Found out the icons now support fill via css feature settings;
`font-variation-settings: 'FILL' 1;`

**Reply by rtcpw** [reply]
This does not work with SVG material icons.

**Comment by damvaz** [comment]
```css
.q-icon {
  font-variation-settings:
    'FILL' 1, // 0 = off; 1 = on
    'wght' 400,
    'GRAD' 0,
    'opsz' 20;
}
````

Suggestion: use css custom props or sass variables for a clean control ✨

**Comment by hawkeye64** [comment, marked as answer]
https://github.com/quasarframework/quasar/pull/18399
